sceptre/template_handlers/http.py

```diff
@@ -4,10 +4,19 @@
 import tempfile
 import sceptre.template_handlers.helper as helper
 
+from requests.adapters import HTTPAdapter
+from requests.exceptions import InvalidURL, ConnectTimeout
+from requests.packages.urllib3.util.retry import Retry
 from sceptre.exceptions import UnsupportedTemplateFileTypeError
 from sceptre.template_handlers import TemplateHandler
 from urllib.parse import urlparse
 
+HANDLER_OPTION_KEY = "http_template_handler"
+HANDLER_RETRIES_OPTION_PARAM = "retries"
+DEFAULT_RETRIES_OPTION = 5
+HANDLER_TIMEOUT_OPTION_PARAM = "timeout"
+DEFAULT_TIMEOUT_OPTION = 5
```

Comment:
5 seems a bit low, I guess. But I also can't see why something as simple as a text document like a template should take 5 seconds to download. 🤷

Reply:
yeah, i've noticed that it's mostly just getting the connection that fails occasionally not actually downloading the file.